mono/tests/bug-gh-17285.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+
+namespace ConsoleApp2
+{
+    class Program
+    {
+        class C
+        {
+            public void f(int i1, int i2, int i3, int i4, int i5, int i6, int i7)
```

Comment:
Can we rename some of these things to make it clearer what we are testing? Like "MethodThatTriggersX86CallFrameOptimization" or something like that?

Reply:
I just added a comment that should help clarify what this is testing.